Src/ResourceUploadBatch.cpp

```diff
@@ -42,10 +42,16 @@ namespace
 #   include "Shaders/Compiled/GenerateMips_main.inc"
 #endif
 
-    bool FormatIsUAVCompatible(DXGI_FORMAT format)
+    bool FormatIsUAVCompatible(_In_ ID3D12Device* device, bool typedUAVLoadAdditionalFormats, DXGI_FORMAT format)
     {
         switch (format)
         {
+        case DXGI_FORMAT_R32_FLOAT:
+        case DXGI_FORMAT_R32_UINT:
+        case DXGI_FORMAT_R32_SINT:
+            // Unconditionally supported.
+            return true;
```

Comment:
It's possible the shader won't bind without ``typedUAVLoadAdditionalFormats`` set, but I don't know I've actually encountered a system that would express that problem.

Reply:
Actually I found a machine with older drivers that exercised this path. It works, but of course only for R32_FLOAT, R32_UINT, or R32_SINT which isn't particularly useful.